### PR TITLE
Discover page 

### LINKED ISSUE #6: create list page to discover movies and tv shows

### PR DESCRIPTION
**Pull request by shailja-imw**
=> card design

closes #6

### CODE REVIEW

**Comment by shailja-imw** [comment]
Design
Nuxt3

**Comment by shailja-imw** [comment]
image

File: app/assets/css/main.css [hunk 1]
```diff
@@ -49,6 +51,7 @@
 
 .main-content {
   margin-top: 84px;
+  margin-bottom: 244px;
```
Comment (monikagorkhiya) [inline app/assets/css/main.css:54]: ?? why need this huge number pixel to margin ?

File: app/pages/index.vue [hunk 2]
```diff
@@ -8,13 +8,5 @@ definePageMeta({
...
 </script>
 <template>
-  <div class="text-center homepage vh-100">Welcome to nuxt3</div>
-  <div class="text-center homepage vh-100">Welcome to nuxt3</div>
-  <div class="text-center homepage vh-100">Welcome to nuxt3</div>
+  <div>Welcome</div>
 </template>
-<style scoped>
-.homepage {
-  margin-top: 400px;
```
Comment (monikagorkhiya) [inline app/pages/index.vue:17]: Same here why need a custom margin?

File: app/pages/discover/index.vue [hunk 3]
```diff
@@ -1,3 +1,36 @@
+<script setup>
+const config = useRuntimeConfig()
+const baseUrl = config.public.base_url
+const { data: showsData, pending } = await useLazyAsyncData('showsData', () =>
```
Comment (monikagorkhiya) [inline app/pages/discover/index.vue:4]: Is that Possible via useAsyncData instead of Lazy?

**Review by monikagorkhiya — CHANGES_REQUESTED** [review]

**Comment by monikagorkhiya** [comment]
Please share Snapshot in the PR

File: app/assets/css/main.css (hunk 1 above)
Comment (shailja-imw) [reply app/assets/css/main.css:54]: As I am using the header in the fixed position that why I have used the margin.

File: app/pages/discover/index.vue (hunk 3 above)
Comment (shailja-imw) [reply app/pages/discover/index.vue:4]: Yes, we can use it. As I was using get API that's why I was using It, as per our discussion I will update it to make it consistent

**Review by monikagorkhiya — APPROVED** [review]

**Pull request merged**